targets/targets.json

```diff
@@ -14793,6 +14793,61 @@
             "3701"
         ]
     },
+    "NUCLEO_G474RE": {
+        "inherits": [
+            "FAMILY_STM32"
+        ],
+        "supported_form_factors": [
+            "ARDUINO",
+            "MORPHO"
+        ],
+        "core": "Cortex-M4F",
+        "config": {
+            "clock_source": {
+                "help": "Mask value : USE_PLL_HSE_EXTC | USE_PLL_HSE_XTAL (need HW patch) | USE_PLL_HSI",
+                "value": "USE_PLL_HSI",
+                "macro_name": "CLOCK_SOURCE"
+            },
+            "lpticker_lptim": {
+                "help": "This target supports LPTIM. Set value 1 to use LPTIM for LPTICKER, or 0 to use RTC wakeup timer",
+                "value": 1
+            },
+            "hse_value": {
+                "help": "HSE default value is 25MHz in HAL",
+                "value": "8000000",
+                "macro_name": "HSE_VALUE"
+            }
+        },
+        "extra_labels_add": [
+            "STM32G4",
+            "STM32G474xx",
+            "STM32G474RE"
+        ],
+        "components_add": [
+            "FLASHIAP"
+        ],
+        "macros_add": [
+            "STM32G474xx",
+            "STM32G474RE",
+            "EXTRA_IDLE_STACK_REQUIRED",
+            "MBED_TICKLESS"
+        ],
+        "overrides": {
+            "lpticker_delay_ticks": 0
+        },
+        "device_has_add": [
+            "ANALOGOUT",
+            "FLASH",
+            "MPU"
+        ],
+        "detect_code": [
+            "0221"
```

Comment:
detect code for NUCLEO_G474RE is 0841,
But I agree this detect_code information i snot used by mbed...

Reply:
Updated, see 9ba1ea101de33e83295675f151297e8582742c57